README.md

```diff
@@ -20,17 +20,19 @@ $ composer require open-telemetry/opentelemetry
 
 ### Examples
 
-You can use the [examples/AlwaysSampleTraceExample.php](https://github.com/open-telemetry/opentelemetry-php/tree/master/examples/AlwaysSampleTraceExample.php) file to test out the reference implementation we have.  This can be easily executed with docker by running `./resources/example-using-docker` from the root of the repository.
+You can use the [examples/AlwaysSampleTraceExample.php](https://github.com/open-telemetry/opentelemetry-php/tree/master/examples/AlwaysSampleTraceExample.php) file to test out the reference implementation we have.  
+First you should start zipkin server by running `docker-compose up -d` and then you can you can easily execute examples using `make example`.
```

Comment:
Maybe this should read something along the lines of, "If you'd like to be able to view example traces, start the zipkin server....."?